public/manifest.json

```diff
@@ -0,0 +1,15 @@
+{
+  "short_name": "React App",
```

Comment:
I'd update all these to whatever you are calling your project

Reply:
This should be updated!